drivers/net/macb.c

```diff
@@ -142,6 +146,12 @@ struct macb_device {
 	unsigned long		pclk_rate;
 #endif
 	phy_interface_t		phy_interface;
+
+#ifdef CONFIG_GPIO
+	struct gpio_desc phy_reset_gpio;
```

Comment:
I have a very strong felling that NET will work without PHY gpio reset - with default settings, so neither this part, neither GPIO driver is not needed to be part of this PR.

Reply:
That depends on default reset state. I have a strong feeling that VPU boot code leaves PHY in reset, but @oleksiimoisieiev can correct me.

Reply:
Unortunately - gpio driver was ported specificaly for this purpose.